velox/functions/prestosql/aggregates/tests/CovarianceAggregationTest.cpp

```diff
@@ -163,6 +163,23 @@ TEST_P(CovarianceAggregationTest, allSameValue) {
   testDistinctGroupBy(aggName, data);
 }
 
+TEST_P(CovarianceAggregationTest, allSameYValue) {
+  vector_size_t size = 1'000;
+  auto data = makeRowVector({
+      makeFlatVector<int32_t>(size, [](auto row) { return row % 7; }),
+      makeFlatVector<float>(size, [](auto row) { return 1; }),
```

Comment:
'row' argument in lambda is not used; linter will complain; please, comment it out

Would you confirm that this test fails without the code changes?

Reply:
The makeFlatVector function requires a lambda expression with one parameter to be passed in.

Reply:
Here is what you need to do:

```
makeFlatVector<float>(size, [](auto /*row*/) { return 1; }),
```